chart/kubedb-catalog/templates/mysql.yaml

```diff
@@ -73,7 +73,7 @@ metadata:
   labels:
     {{- include "kubedb-catalog.labels" . | nindent 4 }}
 spec:
```

Comment:
It's better to deprecate `5.7-v1`, as it does not contain essential changes for group replication.

Reply:
ok